inference/core/interfaces/camera/entities.py

```diff
@@ -96,4 +99,48 @@ def initialize_source_properties(self, properties: Dict[str, float]):
         pass
 
 
+class WebRTCVideoFrameProducer(VideoFrameProducer):
+    def __init__(self, to_inference_queue: deque, to_inference_lock: Lock):
```

Comment:
since this is in the module where the stable code is, could we add our `@experimental` decorator? 
Not sure if that would work on `__init__`, but that would be perfect